### ISSUE TITLE
Incorrect flight height distribution for little gull

### ISSUE THREAD

**Issue by RossBower1** [dmpstats/stochCRM#32]
Hi - I'm trying to run the sCRM for little gull, but it seems that the embedded flight height data do not match with the Johnston 2014 data provided by BTO (https://www.bto.org/sites/default/files/u28/downloads/Projects/Final_Report_SOSS02_FlightHeights2014.xls). This results in an overestimate of collision mortality. 
Please can you advise?

**Comment by statsgeeknz** [comment]
Hi Ross.
I've had a quick look. Yes, the one you've indicated is more diffuse - about 48% of the density in the one in stochCRM is in the first 3m, about double what is in the one you've provided. Not sure that leads immediately to higher collisions as it would depend on the lower reach of the turbine blade - you actually might have lower collisions.

Nonetheless, they're not the same. The ones provided in stochCRM are those that were found in Masden 2015, which this is based on. I'm not sure how these were truly derived - Aonghais Cook from the BTO was involved in that development phase and might know more. Note, there are also the bootstraps attributable to Johnston as well - I took the liberty of examining these, and they're more consistent with the FHD in stochCRM, than the spread-sheet you've provided. So there is a mismatch there also.

Short version - I'm not sure of the where these all come from and are outside software devel remit. The ones in stochCRM are "correct" insofar as they *are* the ones from Masden 2015, and further seem consistent with Johnston's bootstrap distributions (which are very uncertain BTW from my eyeballing).

Also, just a note that this software is quite out-of-date from our perspective (superseded by sCRM and the stochLAB R package) - so issues don't get dealt with very quickly, as this is effectively a community service now. It is open source though :) so feel free to dig into the code and offer a pull request.

C

**Comment by RossBower1** [comment]
Many thanks. Do you know where/if it is possible to obtain the bootstrap data for the Johnston distributions? We haven't been able to locate these.

**Comment by statsgeeknz** [comment]
Hi Ross.
They're contained within stochCRM. You can see them in the repo in the data folder - they're named by species. However, when you select that species in the app, that set of bootstraps will be the one accessed anyway.

BTW - if you wanted to use the other distribution you had in your spreadsheet, you could upload this as a bootstrap. If it is the same distribution, just repeated to conform to the required bootstrap dimensions, you'd be effectively using your own distribution without uncertainty.

C
